staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -194,10 +194,13 @@ type Config struct {
 	// values below here are targets for removal
 	//===========================================================================
 
-	// PublicAddress is the IP address where members of the cluster (kubelet,
+	// AdvertiseAddress is the IP address where members of the cluster (kubelet,
 	// kube-proxy, services, etc.) can reach the GenericAPIServer.
 	// If nil or 0.0.0.0, the host's default interface will be used.
```

Comment:
These two fields will be published as the endpoint of the kubernetes service, correct? Can we state that here?